apps/mobile/src/contexts/SanityDataContext.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  createContext,
+  ReactNode,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { useReportError } from 'shared/contexts/ErrorReportingContext';
+import { fetchSanityContent } from 'src/utils/sanity';
+import { useEffectOnAppForeground } from 'src/utils/useEffectOnAppForeground';
+
+export type MintProject = {
+  name: string;
+  internalId: string;
+  startDate: string;
+  endDate: string;
+  highlightProjectId: string;
+  previewImageUrl: string;
+  artistName: string;
+  collectionName: string;
+  title: string;
+  description: string;
+  funFacts: string[];
+  active: boolean;
+};
+
+type sanityDocumentTypes = {
+  mintProjects: MintProject[];
+};
+
+type SanityDataContextType = {
+  data: sanityDocumentTypes | null;
+};
+
+const SanityDataContext = createContext<SanityDataContextType | undefined>(undefined);
+
+export function useSanityDataContext() {
+  const value = useContext(SanityDataContext);
+  if (!value) {
+    throw new Error('Tried to use SanityDataContext without a Provider');
+  }
+  return value;
+}
+
+export default function SanityDataProvider({ children }: { children: ReactNode }) {
```

Comment:
the only issue i have with this is the naming. we're introducing two new contexts: `AnnouncementContext` and `SanityDataProvider`. on its face it's not entirely clear what different responsibilities they have

how about something like:
- `AnnouncementContext -> `SanityAnnouncementContext`
- `SanityDataContext` -> `SanityMintProjectInfoContext`